lib/shared/image_preview.dart

```diff
@@ -111,7 +111,7 @@ class _ImagePreviewState extends State<ImagePreview> {
             width: widget.width ?? MediaQuery.of(context).size.width - 24,
             fit: BoxFit.cover,
             cache: true,
-            clearMemoryCacheIfFailed: false,
+            clearMemoryCacheWhenDispose: false,
```

Comment:
This fixes an unintended change from here:

https://github.com/thunder-app/thunder/pull/463/files#diff-f771bff454f9ff659025f245c61fbecc7e01a6be62d0ef4c21ada798dd536851R103